### DISCUSSION TITLE
How do I sideload (i.e. install) from a CD/directory of source files outside of Steam/Epic etc.?

### DISCUSSION THREAD

**Discussion by cdrewing** [Heroic-Games-Launcher/HeroicGamesLauncher#2583] [Q&A]
In [these release notes](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.5.0) it says:

> Sideload any app or Game, native or not
> 
> Another big feature that we have on this release is the ability to add any game or application on your computer that are not in the GOG or Epic games account. In short, now you can use Heroic even without an account in these stores.
> On Linux and macOS you can add both native apps or binaries like Scripts, AppImages, etc. And of course windows apps and games and use them with Wine, Proton, or Crossover.
> 
> It is pretty simple to use as everything else in Heroic. Just click on ADD GAME in the library and fill out the form. After writing the title Heroic will get an image automatically from SteamGridDB (thanks @mirkobrombin for help with the API), **you can even run the installer first and select the executable after it.**

This lets open some unanswered questions. Let's say I got a bunch of files on a CD or directory with a setup.exe. So what do I do next? "Select executable" will point to the already installed game *.exe file, right? When selecting the "Run installer first"-button it just shows a file selection dialogue and when I select my setup.exe nothing happens. I am running on Ubuntu MATE 22.04.

Thank you all.

**Comment by gbdrbob** [comment]
> When selecting the "Run installer first"-button it just shows a file selection dialogue and when I select my setup.exe nothing happens.

That's not what should occur when you select an exe with "Run installer first", on my Steam Deck using the flatpak version it launches the installer within the game's wine prefix. If you're still having this problem I'd report it as a bug.
